Source/Core/Core/HW/SI/SI_DeviceGBA.cpp

```diff
@@ -41,21 +41,13 @@ enum EJoybusCmds
   CMD_WRITE = 0x15
 };
 
-const u64 BITS_PER_SECOND = 115200;
-const u64 BYTES_PER_SECOND = BITS_PER_SECOND / 8;
-
-u8 GetNumConnected()
-{
-  int num_ports_connected = s_num_connected;
-  if (num_ports_connected == 0)
-    num_ports_connected = 1;
-
-  return num_ports_connected;
-}
+constexpr auto BITS_PER_SECOND = 115200;
+constexpr auto BYTES_PER_SECOND = BITS_PER_SECOND / 8;
+constexpr auto SEND_MAX_SIZE = 5, RECV_MAX_SIZE = 5;
```

Comment:
If you're going to touch these lines, now is a good time to actually fix the values. Although it might break VBA-M, the values are definitely wrong. We'll probably need to get them to fix their side too.